logrotate/CMakeLists.txt

```diff
@@ -21,12 +21,25 @@ SET(ACTIVE_CPP0xx_DIR "Release")
 INCLUDE (${LogRotate_SOURCE_DIR}/Options.cmake)
 INCLUDE_DIRECTORIES(${G3_HEADER_PATH})
 
+IF (MSVC)
+    SET(CMAKE_DEBUG_POSTFIX "_d")
+ENDIF()
+
 #Find all libraries
 
 find_package(Boost COMPONENTS filesystem system REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 
-find_library(G3LOG ${G3LOG_LIBRARY_TYPE} PATHS ${G3_LIBRARY_PATH})
+find_package( ZLIB REQUIRED )
+include_directories( ${ZLIB_INCLUDE_DIRS} )
+
+IF (MSVC)
+   find_library(G3LOG_DEBUG ${G3LOG_LIBRARY_TYPE}_d PATHS ${G3_LIBRARY_PATH_DEBUG})
+   find_library(G3LOG_RELEASE ${G3LOG_LIBRARY_TYPE} PATHS ${G3_LIBRARY_PATH_RELEASE})
```

Comment:
Finally things have calmed down at work and I am starting to look at this again.

Are you planning to do a pull request on the g3log side as well? Otherwise this breaks g3log integration